repo2docker/buildpacks/conda/environment.py-2.7.frozen.yml

```diff
@@ -1,9 +1,10 @@
 # AUTO GENERATED FROM environment.py-2.7.yml, DO NOT MANUALLY MODIFY
-# Frozen on 2018-01-08 13:23:37 UTC
+# Frozen on 2018-01-11 02:55:46 UTC
 name: r2d
 channels:
 - conda-forge
 - defaults
+- conda-forge/label/broken
```

Comment:
My failure to commit this line in #187 is what caused the failure. Looking at the output shows that this *did* fix the py3 env, which had this line, but since I forgot to refreeze py2.7 after I made the change it failed on the py2 test.

Reply:
More eagle eyed than me, twice :) Missed it when I looked at the original PR and this time :) Thanks for the explanation.